core/src/main/java/com/scalar/db/api/Scan.java

```diff
@@ -34,6 +37,7 @@ public class Scan extends Selection {
   private Optional<Key> endClusteringKey;
   private boolean endInclusive;
   private int limit;
+  private final Set<Conjunction> conjunctions;
```

Comment:
Internally, we use a disjunctive set of conjunctions (i.e., as disjunctive normal form) to manage whole conditions.